report.php

```diff
@@ -268,7 +268,7 @@
         if ($id == XAPI_REPORT_ID_ERROR) {
             $response = '';
             if (isset($result->response)) {
-                $response = '<pre>' . print_r(logstore_xapi_decode_response($result->response), true) . '</pre>';
+                $response = '<pre>' . json_encode(json_decode($result->response), JSON_PRETTY_PRINT)  . '</pre>';
```

Comment:
We should use logstore_xapi_decode_response instead of using json_decode or we can delete that function logstore_xapi_decode_response.

Reply:
I've updated that now - didn't realise I had dropped it